libraries/AP_HAL/Device.cpp

```diff
@@ -178,6 +178,10 @@ bool AP_HAL::Device::write_register(uint8_t reg, uint8_t val, bool checked)
     return result;
 }
 
+bool AP_HAL::Device::write_register_fullduplex(uint8_t *send, uint8_t *recv, uint32_t len)
```

Comment:
why add this new API? why not just call transfer_fullduplex()?

Reply:
I was thinking the same as ead_registers. 
I corrected calling it directly.